js/reference.js

```diff
@@ -6389,12 +6425,60 @@
         histogram: function (bucketCount, min, max) {
             verify(typeof bucketCount === "number", "Invalid bucket count type.");
             verify(min !== undefined && max !== undefined, "Minimum and maximum are required.");
+            var width, range, minMoment, maxMoment;
+
+            var absMax = max,
+                moment = module._moment;
+
+            if (this.column.type.rootName.indexOf("date") > -1) {
```

Comment:
Please add comments of why you are doing these changes, it makes it easier to follow the code.